### ISSUE TITLE
Repo unmaintained?

### ISSUE THREAD

**Issue by knocte** [NickHeiner/fp#1]
Hi there, I see this repo a bit unmaintained?

Not to worry, I’m not going to request you to do some bug fix or some feature for me for free. I’m also a dev, and I also contribute to opensource, so I can work on my own.

Thing is, I’m lately starting to learn TypeScript and I’ve found the need for some FP patterns in my code. I’ve tried to find FP-related libraries in the NodeJS ecosystem but none of them I really like that much.

Sure, I could just create my own, but given that the best FP-related names are taken, it would probably never have much traction.

So I was wondering if I could contribute some PullRequests to your repo to start getting interesting FP-style functions to it, to add on top the ones you have implemented? One first example I could add is an Option type (in some languages also called “Maybe”). If you don’t have time to review my PRs these days, just let me know and I would happily take over the project to start adding things myself. If you check my GitHub profile you will see that I’m a very active open source contributor who’s mainly using F#, but these days I need to work on TypeScript.

Please let me know. Thanks!

**Comment by NickHeiner** [comment]
You’re right that I don’t maintain this, and I also gave the npm name away
to someone else years ago. :)

**Comment by knocte** [comment]
Hey @NickHeiner , thanks for getting back to me.

Are you sure? In npmjs.com (https://www.npmjs.com/package/fp) it is still you, and only you, listed in the "collaborators" list: https://www.npmjs.com/~nick.heiner

**Comment by knocte** [comment]
@NickHeiner ping
